### ISSUE TITLE
Investigate impossible sedes

### ISSUE THREAD

**Issue by whoopsedesy** [sasansom/sedes#40]
The corpus contains a small number of words with an impossible sedes of 9.5, 13, 14, 15, or undefined ("NA"). See what is causing them.

![sedes-dist all](https://user-images.githubusercontent.com/53617134/108256399-68d6d280-7155-11eb-98fe-bda68b1944e2.png)

```
         work book_n line_n word_n      word    lemma sedes metrical_shape scanned num_scansions
221757  Dion.     15    229      1     χώδεα    χώδεα    NA           <NA>    auto             0
342389    Od.     10    243      5      αἰὲν      ἀεί   9.5             +-  manual             1
447180   Q.S.     13     61      1 δυσμενέας δυσμενής    NA           <NA>    auto             0
447181   Q.S.     13     61      2       τοὶ        ὁ    NA           <NA>    auto             0
447182   Q.S.     13     61      3        δ’       δέ    NA           <NA>    auto             0
447183   Q.S.     13     61      4       ἔτ’     ἔτης    NA           <NA>    auto             0
447184   Q.S.     13     61      5   ἔρεσσον   ἐρέσσω    NA           <NA>    auto             0
447185   Q.S.     13     61      6       ἔσω      ἔσω    NA           <NA>    auto             0
447186   Q.S.     13     61      7      ἁλός      ἅλς    NA           <NA>    auto             0
447187   Q.S.     13     61      8        αἱ        ὁ    NA           <NA>    auto             0
447188   Q.S.     13     61      9        δ’       δέ    NA           <NA>    auto             0
447189   Q.S.     13     61     10  ἐφέροντο     φέρω    NA           <NA>    auto             0
467098 Theoc.     15     15      1     καλὸς    καλός  13.0             --  manual             1
467099 Theoc.     15     15      2     ἀπφῦς    ἀπφῦς  14.0             ++  manual             1
467522 Theoc.     15     73      1     ὄχλος    ὄχλος  13.0             ++  manual             1
467523 Theoc.     15     73      2     ἄθρως    ἄθρως  15.0             ++  manual             1
```

Program used to get the above:

```
# Rscript impossible-sedes.R corpus/*.csv
PERMISSIBLE_SEDES = c(1, 2, 2.5, 3, 4, 4.5, 5, 6, 6.5, 7, 8, 8.5, 9, 10, 10.5, 11, 12)
data <- data.frame()
for (csv_filename in commandArgs(trailingOnly=TRUE)) {
	data <- rbind(data, read.csv(csv_filename, na.strings=c(""), stringsAsFactors=FALSE))
}
options(width=200)
data[!(data$sedes %in% PERMISSIBLE_SEDES), ]
```

**Comment by sasansom** [comment]
221757    Dion.     15    229      1     χώδεα --- the manuscript is corrupt at this line, and χώδεα is the only legible word. Rouse, the editor of the 1940 Loeb, seems to suggest that the word appears at or near sedes 9. Thus I've scanned it a single word in known.py preceded by four dactyls, followed by a spondee: (('', '+--'), ('', '+--'), ('', '+--'), ('', '+--'), ('χώδεα', '+--'), ('', '++')).

342389    Od.     10    243      5      αἰὲν --- this line was incorrectly scanned in known.py and has now been corrected.

447180–447189    Q.S.     13     61 --- Hopkinson, the editor of the 2018 Loeb, suggests this line was originally two separate lines. I've now scanned it as one line in known.py.

467098–467099    Theoc.     15     15 --- this is the latter section of a split line which had been incorrectly scanned in known.py and has now been corrected.

467522–467523    Theoc.     15     73 --- same issue and solution as Theoc. 15.15 above.
